src/wazuh_modules/wm_database.c

```diff
@@ -450,8 +448,16 @@ void sync_agents_artifacts_with_wdb() {
             char *agent_name = NULL;
             if (agent_id > 0 && (agent_name = wdb_get_agent_name(agent_id, &wdb_wmdb_sock)) != NULL) {
                 if (*agent_name == '\0') {
+                    // Getting name from database file
```

Comment:
Maybe adding the database filename pattern into the comment will improve the understanding of the next code lines

Reply:
Added!